### PR TITLE
handle no version in raml

### PR DESCRIPTION
**Pull request by samskiter**
The api i'm working with doesn't have 'v1' in the url. seeing as the URLs aren't versioned, i figured nothing should be. I set the RAML version to nothing and had a couple of issues, simple fixes...

### CODE REVIEW

File: lib/retrofit-client/index.js [hunk 1]
```diff
@@ -94,10 +94,12 @@ generator.parser = function(data) {
   }
   model = {};
   model.methods = methodParse;
-  model.version = data.version;
+  model.version = data.version ? "/" + data.version : "";
   if (data.extra) {
-    data.extra["package"] = "" + data.extra["package"] + "." + data.version;
-    data.extra.importPojos = "" + data.extra.importPojos + "." + data.version;
+    if (data.version) {
```
Comment (MachinesAreUs) [inline lib/retrofit-client/index.js:99]: maybe like this

``` javascript
if (data.extra && data.version)
```

Comment (samskiter) [inline lib/retrofit-client/index.js:99]: what about `model.extra = data.extra` ?

**Comment by samskiter** [comment]
hey @atomsfat are these projects still alive?

**Comment by atomsfat** [comment]
@samskiter I released this 0.0.4

**Comment by samskiter** [comment]
Thanks very much both :+1:

**Pull request merged**